dev/org.eclipse.php.target.current/org.eclipse.php.target.current.target

```diff
@@ -13,14 +13,19 @@
 
 <unit id="org.eclipse.ptp.remote.server.core" version="" />
 <unit id="org.eclipse.remote.feature.group" version="0.0.0"/>
-<unit id="org.eclipse.tm.terminal.feature.feature.group" version="0.0.0"/>
-
 
 <unit id="org.eclipse.zest.core" version="0.0.0"/>
 <unit id="org.eclipse.zest.layouts" version="0.0.0"/>
 <unit id="org.eclipse.wildwebdeveloper.feature.feature.group" version="0.0.0"/>
 <repository location="https://download.eclipse.org/releases/2025-06/"/>
 </location>
+
```

Comment:
Will be better to just update main location from 2025-06, also in main pom <eclipse.name> for eclipse-current profile should be updated

Reply:
Done!